tests/end_to_end/test_client_json.py

```diff
@@ -28,17 +26,54 @@ def setUp(self):
         self._backend = backend.FileSystemBackend(self._dataDir)
         self._client = client.LocalClient(self._backend)
 
+    def captureCliOutput(self, command, arguments, outputFormat):
+        clientCommand = "{} {} {} -O {}".format(
+            command, self._dataUrl, arguments, outputFormat)
+        stdout, stderr = utils.captureOutput(
+            cli.client_main, clientCommand.split())
+        self.assertEqual(len(stderr), 0)
+        return stdout
+
+
+class TestClientFasta(TestClientOutput):
+    """
+    Tests client FASTA output
+    """
+    def captureFastaOutput(self, command, arguments=""):
+        stdout = self.captureCliOutput(command, arguments, "fasta")
+        lines = stdout.split()
+        return lines
+
+    def testListReferenceBases(self):
+        referenceSetIterator = self._client.searchReferenceSets()
+        referenceSet = referenceSetIterator.next()
```

Comment:
Calling ``.next`` is Python 2 only --- the Python 2/3 way to do it is to call ``next(referenceSetIterator)`` instead.